hudi-common/src/main/java/org/apache/hudi/common/table/view/RocksDbBasedFileSystemView.java

```diff
@@ -157,8 +157,7 @@ protected boolean isPartitionAvailableInStore(String partitionPath) {
 
   @Override
   protected void storePartitionView(String partitionPath, List<HoodieFileGroup> fileGroups) {
-    LOG.info("Resetting and adding new partition (" + partitionPath + ") to ROCKSDB based file-system view at "
-        + config.getRocksdbBasePath() + ", Total file-groups=" + fileGroups.size());
+    LOG.info("Resetting and adding new partition ({}) to ROCKSDB based file-system view at {}, Total file-groups={}", partitionPath, config.getRocksdbBasePath(), fileGroups.size());
```

Comment:
would be changed to two lines.

Reply:
In the previous modification, I used two lines, but they were not adopted. I read our verification rules is `<property name="max" value="200"/>`, I'm a little confused now.